swap_meet/clothing.py

```diff
@@ -0,0 +1,13 @@
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "", condition = 0):
+        # super().__init__(category = "Clothing")
+        self.category = "Clothing"
+        self.condition = condition
+
+    def __str__(self):
+        return "The finest clothing you could wear."
+
+    def condition_description(self):
+        return super().condition_description()
```

Comment:
Clothing inherits all of the methods from Item, so removing lines 12 & 13 will result in the same behavior (calling `condition_description` on an instance of Clothing will call the Item method `condition_description`).